std_msgs/String unique_id
Base base_component 
Arm arm_component
